Python_Backend/StateGenerator.py

```diff
@@ -0,0 +1,26 @@
+import pickle
+import pandas as pd
+class StateGenerator:
+    def __init__(self, model_path, model_type):

```

Comment:
I envisioned the classes besides the main "socket in" class just having static functions instead of creating an object. What do you think about this, might warrant a discussion.

Reply:
Per discussion on slack, we are going to keep this as an object. Rework can be done if we find instantiation is causing problems.